src/components/SetPauseCard.jsx

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link';
+
+import useStore from '../hooks/useStore';
+
+import SetPauseIcon from './Icons/SetPauseIcon';
+import StyledCard from './StyledCards';
+
+export default function SetPauseCard() {
+	const setPauseTimes = useStore(state => state.setPauseTimes);
+	const length = setPauseTimes.length;
```

Comment:
I would rename the variables because it's a bit confusing.
-> `const  = pauseTimesLength` 
You shouldn't name a variable starting with `set` because it is a predefined word for a function.

Reply:
done